src/image.tsx

```diff
@@ -148,7 +169,7 @@ export const ImageContent = React.forwardRef<HTMLImageElement, ImageProps>(
         const [isLoading, setIsLoading] = React.useState(true);
         const [hideLoadingFallback, setHideLoadingFallback] = React.useState(false);
 
-        const ratio = typeof aspectRatio === 'number' ? aspectRatio : RATIO[aspectRatio];
+        const ratio = props.circular ? 1 : typeof aspectRatio === 'number' ? aspectRatio : RATIO[aspectRatio];
```

Comment:
what about now with <Circle backgroundImage />? are we support both ways to use circular images in Mística?

Reply:
The problem with `<Circle backgroundImage />` is that we won't have the skeleton (loading) and fallback (error) cases that we have in Image/Video

Reply:
mmm yep, that's my point. Should we evolve Circle to allow this behavior? I understand your approach but now we have two elements in Mística (Circle + Image Circle) with the same usage, is it a problem to have 2 elements to get the same result? When do I have to use circle with image or image circular?

Reply:
there is another difference, [you can have a children in Circle](https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8BhAlgJzAGxgAgCMBDMAawHMMIBXAOygEkBbI8mAXgB0QALAFz4AHAM7wA9GMFoww6kwB0gnhD4RhYgIwAmAAw7NundzzC0ALw7AAbABYAvgD5OtPHiQAVGAA8%2BNvExgoNDkHAAkYbGwIJDFPHxsnWhj0LFxEkAAaED4eGADhBABtEABZCAA3NGE%2BIgxMkAA1NHKIEABdLIB3NCgcgvhCgGYrHTa7IA)

But If we don't think this is a useful feature, I think we could remove/deprecate backgroundImage from Circle

Reply:
that's a good difference. I didn't take it into account, we can keep supporting both cases